src/ssvc/decision_points/exploitation.py

```diff
@@ -40,14 +46,28 @@ def _strip_spaces(s):
     version="1.0.0",
     values=(
         EXP_NONE,
-        POC,
+        POC_1,
+        ACTIVE,
+    ),
+)
+
+EXPLOITATION_2 = SsvcDecisionPoint(
+    name="Exploitation",
+    description="The present state of exploitation of the vulnerability.",
+    key="E",
+    version="1.1.0",
+    values=(
+        EXP_NONE,
+        POC_2,
         ACTIVE,
```

Comment:
Here I added a new version of the Exploitation decision point that picks up the new POC_2 value.

Just noticing I should have named it EXPLOITATION_1_1_0. I'll fix that in another commit.